packages/backend/src/services/ProjectService.ts

```diff
@@ -61,20 +83,121 @@ export class ProjectService {
         return project;
     }
 
-    async runQuery(sql: string): Promise<Record<string, any>> {
-        return this.projectAdapter.runQuery(sql);
+    async startAdapter(projectUuid: string): Promise<ProjectAdapter> {
+        const activeAdapter = this.projectAdapters[projectUuid];
+        if (activeAdapter === undefined) {
+            const project = await this.projectModel.get(projectUuid);
+            const adapter = projectAdapterFromConfig(project.dbtConnection);
```

Comment:
Adapters are now built directly from the ProjectModel not from the config service